packages/client/tests/functional/upsert-relation-mode-prisma/test.ts

```diff
@@ -0,0 +1,19 @@
+import testMatrix from './_matrix'
+// @ts-ignore
+import type { PrismaClient } from './node_modules/@prisma/client'
+
+declare let prisma: PrismaClient
+
+testMatrix.setupTestSuite((_suiteConfig, _suiteMeta) => {
+  test('calling upsert two times in a row does nothing', async () => {
+    for (let i = 0; i < 2; i++) {
+      const node = await prisma.user.upsert({
```

Comment:
```suggestion
      const node = await prisma.node.upsert({
```

Sorry, missed that in original review